File: go/fizzbuzz.go
package main

import(
	"fmt"
	"strconv"
	)

func fizzbuzz(num int) string{

	fizz := num % 3
	buzz := num % 5

	if fizz == 0 && buzz == 0 {
	    return "fizzbuzz"
	} else if fizz == 0 {
		return "fizz"
	} else if buzz == 0 {
		return "buzz"
	} else {
		return strconv.Itoa(num)
	}
}

func main() {
	for i := 1; i <= 31; i++ {
		fmt.Println(fizzbuzz(i))
	}

}

File: php/fizzbuzz.php
<?
$ret_array = array();
for($i=1; $i <= $argv[1]; $i++) {
	if(isFizz($i) && isBuzz($i)) {
		$ret_array[] = "fizzbuzz";
	} else if (isBuzz($i)) {
		$ret_array[] = "buzz";
	} else if (isFizz($i)) {
		$ret_array[] = "fizz";
	} else {
		$ret_array[] = $i;
	}
}
print implode(", ", $ret_array);

function isFizz($num) {
	if ($num % 3 == 0) {
		return true;
	}
	return false;
}

function isBuzz($num) {
	if($num % 5 == 0) {
		return true;
	}
	return false;
}

?>